index.js

```diff
@@ -163,7 +163,7 @@ const headers = {
 		await updateCheck(id, {
 			status: 'completed',
 			completed_at: (new Date()).toISOString(),
-			conclusion: errorCount > 0 ? 'failure' : 'success',
+			conclusion: errorCount > 0 ? 'failure' : warningCount > 0 && 'neutral' || 'success',
```

Comment:
This is great! : +1:

Might read better like this:
```suggestion
			conclusion: errorCount > 0 
				? 'failure' 
				: warningCount > 0 
					? 'neutral' 
					: 'success',
```

Reply:
eslint rules disallow nested ternary...

Reply:
:cry: